Penteract/Assets/sphere2.fbx.meta

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think you used this as an early mockup when we were doing tests, please, remove it